sdl2promela/sdl/model.py

```diff
@@ -15,14 +16,40 @@ def __init__(self):
         self.name = None
 
 
+class Expression:
+    """Base class for expressions."""
+
+    pass
+
+
+class Constant(Expression):
+    """Constant value."""
+
+    value: str
```

Comment:
string? What about other types?

Reply:
Now everyting is a string, and IMHO this is good enough for now. However, this may change when support for complex types is added.